app/views/orders/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="cart-screen">
+    <div class="carrito-title">
+        <h2>Your order</h2>
+    </div>
+    <div class="card-product-detail-cart">
+            <div class="cart-rows-container">
+                <li class="cart-rows my-order-cart-rows">ID</li>
+                <li class="cart-rows my-order-cart-rows">PHONE</li>
+                <li class="cart-rows my-order-cart-rows address-cart-row">ADDRESS</li>
+                <li class="cart-rows my-order-cart-rows">PRODUCT</li>
+                <li class="cart-rows cart-rows-action quantity-cart-row">QUANTITY</li>
+            </div>
+            <div class="line_items content-my-order">
+                <% @order.order_items.each do |order| %>
+                    <li class="cart-rows-content bold-cart-row my-order-cart-content id-cart-row"><%= @order.id %> </li>
+                    <li class="cart-rows-content my-order-cart-content"><%= @order.phone %> </li>
+                    <li class="cart-rows-content my-order-cart-content address-cart-order"><%= @order.address %> </li>
```

Comment:
you don't need to shoe the phone and adress for each order item as they have all the same, the address and phone are order attributes (not order item attributes) so you need to show it just once in this view